Docker/Dockerfile

```diff
@@ -87,11 +101,11 @@ LABEL author="Xiaobo Tian <peterwillcn@gmail.com>" \
   description="This is an eosio/eos image"
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install openssl && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /usr/local/lib/* /usr/local/lib/
-COPY --from=builder /opt/eos/bin /opt/eos/bin
+COPY --from=builder /tmp/build/install/bin /opt/eosio/bin
 COPY --from=builder /tmp/build/contracts /contracts
 COPY --from=builder /eos/Docker/config.ini /eos/genesis.json /
-COPY start_eosd.sh /opt/eos/bin/start_eosd.sh
-RUN chmod +x /opt/eos/bin/start_eosd.sh
+COPY start_eosiod.sh /opt/eosio/bin/start_eosiod.sh
+RUN chmod +x /opt/eosio/bin/start_eosiod.sh
 ENV LD_LIBRARY_PATH /usr/local/lib
-VOLUME /opt/eos/bin/data-dir
-ENV PATH /opt/eos/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+VOLUME /opt/eosio/bin/data-dir
+ENV PATH /opt/eosio/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Probably best to stay with `eos` instead of `eosio` on the dawn2.x branch to avoid confusion.